libs/base/ui/form/input-list/src/input-list.styles.ts

```diff
@@ -36,3 +37,14 @@ export const styles = css`
     margin-inline-start: 1px;
   }
 `;
+
+export const screenStyles = screenCss({
+  sm: css`
+    legend {
+      margin-block-end: 20px;
+      font-weight: 500;
+      font-size: 14px;
+      line-height: 22px;
```

Comment:
Both large and small screens should use standard body typography. Let's use design tokens here.